test/powershell/Language/Scripting/CheckRestrictedlanguage.Test.ps1

```diff
@@ -0,0 +1,120 @@
+﻿#  <Test>
+#    <summary>Test restricted language check method on scriptblocks</summary>
+#  </Test>
+Describe "Test restricted language check method on scriptblocks" -Tags "CI" {
+        BeforeAll {
+            set-strictmode -v 2
+        }
+
+        function list {
+
+            $l = new-object system.collections.generic.list[string]
+            $args | foreach {$l.Add($_)}
+            , $l
+        }
+
+        It 'Check basic expressions' {
+
+            {2+2}.CheckRestrictedLanguage($null, $null, $false)  # Succeed with no variables
```

Comment:
is there supposed to be a should statement here?
`{ {2+2}.CheckRestrictedLanguage($null, $null, $false)} | Should Not Throw`